testsuite/src/it/java/org/graalvm/tests/integration/utils/BuildAndRunCmds.java

```diff
@@ -50,6 +54,40 @@ public enum BuildAndRunCmds {
             new String[]{"mvn", "dependency:sources", "-Dquarkus.version=" + QUARKUS_VERSION.getVersionString()},
             new String[]{IS_THIS_WINDOWS ? "target\\quarkus-runner.exe" : "./target/quarkus-runner"}
     }),
+    QUARKUS_JSON_PERF_PARSEONCE(new String[][]{
+            // TODO tune and report: https://www.graalvm.org/22.0/reference-manual/native-image/MemoryManagement/
+            new String[]{"mvn", "clean", "package", "-Pnative", "-Dquarkus.version=" + QUARKUS_VERSION.getVersionString(),
+                    "-Dquarkus.native.additional-build-args=" +
+                            ((UsedVersion.getVersion(false).compareTo(Version.create(23, 0, 0)) >= 0) ? "-H:+AllowDeprecatedBuilderClassesOnImageClasspath," : "") +
+                            "-R:MaxHeapSize=" + MX_HEAP_MB + "m," +
+                            "-H:-ParseOnce," +
+                            "-H:BuildOutputJSONFile=quarkus-json_minus-ParseOnce.json",
+                    "-Dfinal.name=quarkus-json_-ParseOnce"},
+            new String[]{"mvn", "package", "-Pnative", "-Dquarkus.version=" + QUARKUS_VERSION.getVersionString(),
+                    "-Dquarkus.native.additional-build-args=" +
+                            ((UsedVersion.getVersion(false).compareTo(Version.create(23, 0, 0)) >= 0) ? "-H:+AllowDeprecatedBuilderClassesOnImageClasspath," : "") +
+                            "-R:MaxHeapSize=" + MX_HEAP_MB + "m," +
+                            "-H:+ParseOnce," +
+                            "-H:BuildOutputJSONFile=quarkus-json_plus-ParseOnce.json",
+                    "-Dfinal.name=quarkus-json_+ParseOnce"},
```

Comment:
I suggest avoiding the duplication here since only `ParseOnce` changes, to make it easier to keep the rest the same.

Reply:
I will clean it up.